### DISCUSSION TITLE
Implement SL likelihoods

### DISCUSSION THREAD

**Discussion by slosar** [LSSTDESC/firecrown#600] [Ideas]
This is a ticket that is similar to SN likelihood, only that it operates with the time-delay distances, there is no marginalization over the absolute amplitude. Sufficiently different that it needs to be implemented separately, although the implementation can start efficiently as a copy-paste of SN likelihood.
See also ticket: https://github.com/LSSTDESC/firecrown/issues/68

**Comment by heatherprince** [comment]
I had started working on this, based on Tom Collett's DESC SRD likelihood. I was getting some strange results but haven't looked at in a while, I will dig it up and update here!
